post.js

```diff
@@ -34,7 +36,41 @@ async function trimPkgsCacheFolder (cacheFolder) {
   core.endGroup()
 }
 
+async function executeMicromambaShellDeinit (shell, logLevel) {
+  const cmd = micromambaCmd(`shell deinit -s ${shell} -p ${PATHS.micromambaRoot} -y`, logLevel, PATHS.micromambaExe)
+  const cmd2 = cmd.split(' ')
+  return await executeSubproc(cmd2[0], cmd2.slice(1))
+}
+
+const deinitProfile = {
```

Comment:
Maybe this as well but not sure how much harder it will be to understand in deduplicated form